integration_tests/rest_api_utils.py

```diff
@@ -987,6 +987,34 @@ def ensure_llm_task_response_is_correct(
         assert re.search(response_text_regex, response["output"]["text"])
 
 
+def ensure_llm_task_stream_response_is_correct(
+    response: str,
+    required_output_fields: Optional[List[str]],
+    response_text_regex: Optional[str],
+):
+    print("raw response", response)
+    # parse response
+    #  data has format "data: <data>\n\ndata: <data>\n\n"
+    #  We want to get a list of dictionaries parsing out the 'data:' field
+    parsed_response = [
+        json.loads(r.split("data: ")[1]) for r in response.split("\n") if "data:" in r.strip()
+    ]
+
+    print("parsed response: ", parsed_response)
+    # if required_output_fields is not None:
+    #     for field in required_output_fields:
+    #         for response in parsed_response:
+    #             assert field in response["output"]
```

Comment:
should implement these?

Reply:
i'm not really sure what this is supposed to be testing. the response for streaming is different from that for sync so the same checks don't really apply